staging/src/k8s.io/client-go/dynamic/testdata/TestGoldenRequest/create

```diff
@@ -0,0 +1,9 @@
+POST /apis/flops/v1alpha1/namespaces/mops/flips/mips/fin?fieldValidation=warn HTTP/1.1
+Host: example.com
+Accept: application/json
+Accept-Encoding: gzip
+Content-Length: 29
+Content-Type: application/json
+User-Agent: TestGoldenRequest
+
+{"metadata":{"name":"mips"}}
```

Comment:
urg.  So this is definitely going to change then because we'll start adding apiVersion and kind?  If I'm remembering correctly (paging back in), we won't always have the values for apiVersion and kind unless someone has configured the scheme, right?

CBOR ought to be able to send without an apiVersion and kind, right?

Reply:
It should not change as currently written for someone passing an Unstructured with an empty GVK. I just rebased https://github.com/kubernetes/kubernetes/pull/124059 on top of this test to demonstrate.